src/helpers/jwks.js

```diff
@@ -23,7 +23,7 @@ function getJWKS(options, cb) {
       var key;
 
       if (err) {
-        cb(err);
+        return cb(err);
```

Comment:
Why did this change? Big behavior difference but I'm not seeing any other changes to how this is handled.

Reply:
it was actually wrong. see a similar PR here: https://github.com/auth0/idtoken-verifier/pull/14/files
if you don't return (or run the second part in an `else`), the code will actually keep running, which is not the desired effect.